cli/src/config/aliases.toml

```diff
@@ -0,0 +1,6 @@
+[aliases]
+# Placeholder: added by user
+
+[revset-aliases]
+trunk = "latest((present(main) | present(master) | present(root)) & remote_branches())"
```

Comment:
Should we do `latest(remote_branches(main) | remote_branches(master) | root)` instead? Otherwise `main`/`master` resolves the the local branch, which doesn't necessarily point to the same place as the remote branch. The disadvantage is that it can resolve to many commits if there are many remotes, but `latest()` should generally take care of that.

Reply:
Maybe it's time to add string matcher syntax like `remote_branches(literal:main)`? `remote_branches(main)` can also match "maint". I think `mine()` revset would have the same problem.

Reply:
Good point. "maint" is quite likely in practice. I don't think we need to block this PR on it, but we probably should add that somewhat soon. Let me know if you disagree.

I thought about for the `mine` revset, but it doesn't seem like much of a problem in practice there. I'm sure it happens that an email address is a suffix of another email address, but that's probably extremely rare, at least in the same repo.